### PR TITLE
Improves seo metadata for api docs

### PR DESCRIPTION
**Pull request by rhubarbselleven**
Grabs the first paragraph from the doc and uses that as a seo.description. Then fills out the usual open graph and twitter card details.

Tried to figure out howto get the date of the tag (to use for dc.created), but couldn't figure out a reliable process to put into gitty.

### CODE REVIEW

**Comment by rhubarbselleven** [comment]
why travis why!?

**Comment by peterblazejewicz** [comment]
`git commit --amend` and then `git push -f` in your PR? We have used this yesterday to restart Travis

**Comment by samccone** [comment]
yeah something is wrong with my login in my bash file... if someone want to check this out and run lint manually i will merge ... sorry about this will get it sorted soon

**Comment by jasonLaster** [comment]
this is an awesome idea @rhubarbselleven. I can't wait to see @cmaher come in with cool graphs

**Comment by peterblazejewicz** [comment]
I'll test this and merge locally. brb

**Comment by samccone** [comment]
kk sorry about the test stuff

File: src/docs/template.hbs [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
+    <meta property="og:site_name" content="Marionette.js" />
+    <meta property="og:description" content="<%= file.description %>" />
+    <meta property="og:type" content="article" />
+    <meta property="og:url" content="http://www.marionettejs.com/docs/<%- tag + '/' + file.basename %>.html" />
+    <meta property="og:image" content="http://marionettejs.com/images/marionette.png" />
+
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:creator" content="@marionettejs">
+    <meta name="twitter:description" content="<%- file.description %>">
+    <meta name="twitter:image" content="http://marionettejs.com/images/marionette.png">
```
Comment (peterblazejewicz) [inline src/docs/template.hbs]: This one is optional and we should not use it with docs (the same image)
> URL to a unique image representing the content of the page. __Do not use a generic image such as your website logo, author photo, or other image that spans multiple pages.__

_(Twitter docs)_

Comment (peterblazejewicz) [inline src/docs/template.hbs]: to fix later TBD

**Comment by rhubarbselleven** [comment]
Yay :roller_coaster:

File: src/docs/template.hbs (hunk 1 above)
Comment (rhubarbselleven) [inline src/docs/template.hbs]: ew. Alternative would be to edit the svg on the fly lol

**Pull request merged**